src/Control/FunFlow/ContentHashable.hs

```diff
@@ -163,6 +177,8 @@ contentHashUpdate_text ctx (T.Text arr off_ len_) =
       off = off_ `shiftL` 1 -- convert from 'Word16' to 'Word8'
       len = len_ `shiftL` 1 -- convert from 'Word16' to 'Word8'
 
+instance ContentHashable ContentHash where
```

Comment:
Maybe add a 'TODO' note that we should consider output hashing as discussed today?